File: 2525. Categorize Box According to Criteria/Solution.cpp
#include <string>
using namespace std;

class Solution {
public:
    string categorizeBox(int length, int width, int height, int mass)
    {
        bool is_bulky = max(length, max(width, height)) >= 10000 || 1LL * length * width * height >= 1000000000;
        bool is_heavy = mass >= 100;
        if (is_bulky && is_heavy)
            return "Both";
        else if (!is_bulky && !is_heavy)
            return "Neither";
        else if (is_bulky)
            return "Bulky";
        else
            return "Heavy";
    }
};

File: 2525. Categorize Box According to Criteria/Solution.py
class Solution:
    def categorizeBox(self, length: int, width: int, height: int, mass: int) -> str:
        is_bulky = max(length, width, height) >= 10000 or length * width * height >= 1000000000
        is_heavy = mass >= 100
        if is_bulky and is_heavy:
            return "Both"
        elif is_bulky:
            return "Bulky"
        elif is_heavy:
            return "Heavy"
        else:
            return "Neither"
